README.md

```diff
@@ -17,7 +17,6 @@ Build status:
 - Linux:
   [![Linux Build Status](https://travis-ci.org/mozilla/mozregression.svg?branch=master)](https://travis-ci.org/mozilla/mozregression)
   [![Coverage Status](https://img.shields.io/coveralls/mozilla/mozregression.svg)](https://coveralls.io/r/mozilla/mozregression)
-- Windows: [![Windows Build status](https://ci.appveyor.com/api/projects/status/bcg7t1pt2bahggdr?svg=true)](https://ci.appveyor.com/project/wlach/mozregression/branch/master)
```

Comment:
why is windows removed?

Reply:
So the windows build was moved from AppVeyor to GitHub Actions, thus the status can just be seen in the GitHub actions status for each release (or commit). I don't think the Linux Build Status is doing anything right now (not even pointing to the right branch, so this should probably be removed as well).